jr-objects/src/test/java/com/fasterxml/jackson/jr/GroovyTest.groovy

```diff
@@ -0,0 +1,21 @@
+package com.fasterxml.jackson.jr
+
+import com.fasterxml.jackson.jr.ob.JSON
+import com.fasterxml.jackson.jr.ob.TestBase
+
+class GroovyTest extends TestBase {
+
+    void testSimpleObject() throws Exception {
+        var data = JSON.std.asString(new MyClass())
+        var expected = "{\"aDouble\":0.0,\"aStr\":\"stringData\",\"anInt\":0,\"metaClass\":{}}";
```

Comment:
I think ideally we would not serialize this field at all, to avoid problems on deserialization.

Reply:
Now the Meta class will not serialize